rhobs/alerting/data_plane/prometheus.availability_metric_konflux_alerts.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-konflux-up-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: konflux_up_alerts
+    interval: 1m
+    rules:
+    - alert: KonfluxAlert
+      expr: count ((konflux_up offset 1h) unless (konflux_up))
```

Comment:
I think we need to use `... unless on(service, check, source_cluster) ...` because some of the `konflux_up` metric labels like `pod` have significant variability over time and `unless` will compare all labels in each time series by default.

Let's also make sure to include extra, variable labels in the test series to exercise this scenario.

Reply:
Updated

Reply:
I tested it wrong before and thought it's not working if I give `on(service, check, source_cluster)` but updated it now. Thanks for the suggestion @amisstea!